garden-service/test/unit/src/config/base.ts

```
@@ -256,57 +259,6 @@ describe("loadConfig", () => {
     ])
   })
 
-  it("should parse a config file using the flat config style", async () => {
```

Why remove this?

---

Because we no longer support the nested config style, and all configs therefore have the flat style. I mentioned this in the notes above.

---

Right, I just thought the test was still valid, but the description wasn't. So this same test is actually performed elsewhere then?

Actually the PR note says the "flat style" has been removed, which it hasn't. The nested style was removed.

---

All test projects now use the flat style so the tests above implicitly test for what this is test is testing for. But we can leave it in as well and re-name the project and update the test description.

---

...and yeah, the PR note was wrong. My bad.

---

If the other tests cover this, that's fine. But we should keep the one you deleted below, right? Or is there another test for that case specifically?

---

I've added a new test for the case below (`should throw if a config specifies an invalid kind`) and added a `variables` field to the `test-project-multiple-modules` project so that the `should load and parse a config file defining a project and a module` test now covers all the cases the test that I removed covered.

---

Still waiting to push that change? Or am I losing my mind and not seeing it somewhere? (I won't rule that out 😛)

---

Bah, apparently forgot to push. It's been a long week...